resources/public/api/conf/1.0/schemas/BonusPaymentRequest.post.schema.json

```diff
@@ -69,8 +69,28 @@
           "claimReason"
         ]
 
+      },
+      "supersede" :
+      {
+        "description": "Supresede bonus payment by requesting for an additional bonus amount or recover an overpaid amount",
+        "type": "object",
+        "properties" :
+        {
+          "automaticRecoveryAmount":{"description": "This is used to confirm the amount due to HMRC from the LISA manager. If the superseded transaction result is a less than zero, you must give the automatic recovery amount. This is so that HMRC can calculate the amount to collect from the LISA manager. Leave it blank if there are no funds available. HMRC will recover overpaid funds from the LISA investor.", "type" : "number"},
+          "supersededTransactionID" :{"description": "The TransactionID of the original bonus affected by the correction or recovery. This is used together with the superseded transaction amount to make sure the right bonus transaction is replaced.", "type" : "number"},
```

Comment:
Should we use 'transactionId' rather than 'TransactionID'? This PR has a mix of Transaction ID and TransactionID also, we should aim to use the same everywhere.